tasks/sep/019-fp-holocene-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,24 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
```

Comment:
Validation is missing the state overrides section. Let's make sure to add this so signers (especially on mainnet) understand why the state overrides are present. You can pull these from another nested ceremony's validations, they are always identical

Reply:
Could we have a generic nested signer validation doc that can always be referenced instead? This would save a lot of boilerplate and copy-pasting.

Reply:
We are going to improve this process by autogenerating this file as much as we can, but yes definitely feel free to pull it out into a separate file and reference it!

Reply:
Agreed with @mds1 that this will only be addressed in the follow up task to fix the Sepolia FPs with the correct `DelayedWETH` implementation.